content/Software_Projects/Mobile_SDK/index.adoc

```diff
@@ -0,0 +1,29 @@
+== Mobile SDK
+
+=== Get Started with the YubiKey 5Ci
+
+The YubiKey 5Ci is a multi-protocol security key equipped with FIDO2/WebAuthn, FIDO U2F, OTP (one-time password), PIV (Smart Card), and OpenPGP. With support for multiple authentication protocols, the YubiKey 5Ci delivers strong multi-factor, second-factor (2FA), and single-factor passwordless authentication for a simple and seamless user experience across the entire computing ecosystem.
+
+=== Yubico Mobile iOS SDK
+
+The Yubico Mobile iOS SDK is an iOS library provided by Yubico to interact with YubiKeys on iOS devices.
+
+* https://github.com/Yubico/yubikit-ios[Download iOS SDK]
+
+The library supports NFC-enabled YubiKeys and provides the APIs to request an OTP (Yubico OTP or HOTP) from the NFC YubiKeys using a NFC-enabled iOS device. It also provides a built-in QR Code reader which can be used as an alternative enrolment mechanism for iOS devices which don't support NFC reading.
+
+Starting from version 2.0.0, YubiKit adds support for YubiKeys with lightning connector, such as the YubiKey 5Ci, a security key design by Yubico for iOS devices.
```

Comment:
Can we make this the first paragraph?

Reply:
We can but would need clarifying from Lan